eng/.docsettings.yml

```diff
@@ -125,6 +125,8 @@ known_content_issues:
   - ['sdk/textanalytics/azure-ai-textanalytics/swagger/README.md', '#3113']
   - ['sdk/template/azure-sdk-template/README.md','has other required sections']
   - ['sdk/storage/azure-storage-file-share/swagger/README.md','has other required sections']
+  - ['sdk/search/azure-search/src/test/README.md', '#3113']
+  - ['sdk/search/open-api/readme.md', '#3113']
```

Comment:
Is this a module that we ship? If so we should have the readme.

Reply:
yes, it's ready to ship. but those readme files are for running the testing framework and generating the autorest code using swagger (open api) spec